Repo/TodoRepo.cs

```diff
@@ -0,0 +1,73 @@
+using Microsoft.AspNetCore.Mvc;
+using todoDotNet6.Repo;
+
+namespace todoDotNet6.TodoRepo
+{
+
+    public interface ITodoRepository : IRepository<Todo,Guid> {}
+
+    public class TodoRepo : ControllerBase, ITodoRepository
+    {
+        public static Dictionary<Guid, Todo> todos = new Dictionary<Guid, Todo>() {};
+
+        public Todo CreateTodo(Todo request)
+        {
+            request.Id = Guid.NewGuid();
+            request.CreatedDate = DateTime.Now;
```

Comment:
We could making setting `CreatedDate` manually unnecessary with a little C# trick you could totally steal from my repo